px_metadata.json

```diff
@@ -58,6 +59,14 @@
         "test_page_requested_activity_cookie_origin",
         "test_block_page_hard_block_response",
         "test_risk_api_validate_cookie_origin",
-        "test_risk_cookie_valid_cookie_with_user_agent_bigger_than_max_length"
+        "test_risk_cookie_valid_cookie_with_user_agent_bigger_than_max_length",
+        "test_risk_api_schema",
+        "test_block_activity_schema",
+        "test_page_requested_activity_schema",
```

Comment:
You know you can simplify these three lines with this:

```
"test_(risk_api|block_activity|page_requested_activity)_schema"
```

Reply:
changed